### ISSUE TITLE
EmptyStateView is red after building with iOS 14 SDK

### ISSUE THREAD

**Issue by jannemecek** [luispadron/UIEmptyState#33]
Apparently this line was left in
https://github.com/luispadron/UIEmptyState/blob/b0d6ca751a81880257719e198fffe198269ba302/src/UIEmptyState/UIEmptyStateView.swift#L256

Previously the background color wouldn't be shown on UIStackView, which has changed now.

**Comment by RayanSaeed** [comment]
+1

**Comment by gereons** [comment]
I don't think this bug will be fixed anytime soon, this repo appears to be abandoned.

For those with the same issue that do not want to maintain their own fork, an easy workaround is to add

```
func emptyStateViewWillShow(view: UIView) {
        if let emptyView = view as? UIEmptyStateView {
            emptyView.contentView.backgroundColor = .clear
        }
}
```

to your `UIEmptyStateDelegate`
